ignite-base/App/Themes/Colors.js

```diff
@@ -1,7 +1,9 @@
 // @flow
 
-const colors = {
-  background: '#1F0808',
+import { merge } from 'ramda'
+
+const igniteColors = {
+  igniteBackground: '#1F0808',
```

Comment:
background is a common name for the background color of Ignited apps. This enables devs to use 'background' as a color for their own apps without changing the background color of the Ignite-generated screens such as ignite/ignite-base/App/Containers/Styles/ListviewExampleStyle.js